tasks/configure.yml

```diff
@@ -7,7 +7,9 @@
     mode: 0644
     section: "{{ item.section | default(None) }}"
     option: "{{ item.option }}"
-    value: "{% if item.value == true %}true{% elif item.value == false %}false{% else %}{{ item.value }}{% endif %}"
+    value: "{% if item.value | type_debug=='bool' and item.value %}true
+      {% elif item.value | type_debug=='bool' and not item.value %}false
+      {% else %}{{ item.value }}{% endif %}"
```

Comment:
Maybe a bit simpler?

```yaml
    value: "{{ item.value | to_json if item.value | type_debug == 'bool' else item.value }}"
```
The `to_json` converts any boolean value into `true` or `false`, otherwise use whatever the value of `item.value` is (works with `None`).

Reply:
Or even

```yaml
    value: "{{ (item.value | type_debug == 'bool') | ternary(item.value | to_json, item.value) }}"
```
:-)

Reply:
Nice! Trying that now.